### PR TITLE
Add documentation for Verizon Fios Quantum Gateway device_tracker platform

### PR DESCRIPTION
**Pull request by cisasteelersfan**
**Description:**
Documentation for new Verizon Fios Quantum Gateway device_tracker platform. Plus fixing a typo in huawei_router.markdown.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17023

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
👍

File: source/_components/device_tracker.huawei_router.markdown [hunk 1]
```diff
@@ -12,7 +12,7 @@ ha_category: Presence Detection
 ha_release: 0.51
 ---
 
-The `huawei` device trakcer platform offers presence detection by looking at connected devices to a [Huawei router](http://m.huawei.com/enmobile/enterprise/products/network/access/pon-one/hw-371813.htm).
+The `huawei` device tracker platform offers presence detection by looking at connected devices to a [Huawei router](http://m.huawei.com/enmobile/enterprise/products/network/access/pon-one/hw-371813.htm).
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.huawei_router.markdown]: Please don't include unrelated changes. Those can come in a separate PR.

File: source/_components/device_tracker.quantum_gateway.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+title: "Quantum Gateway"
+description: "Instructions on how to integrate Quantum Gateways into Home Assistant."
+date: 2018-09-30 01:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Presence Detection
+logo: fios.svg
+ha_release: 0.80
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.quantum_gateway.markdown]: 0.81

File: source/_components/device_tracker.huawei_router.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/device_tracker.huawei_router.markdown]: I don't agree with that @MartinHjelmare. Yes, it should be limited, but this is just fixing a obvious typo.

@cisasteelersfan Well spotted, thanks.

Comment (frenck) [reply source/_components/device_tracker.huawei_router.markdown]: @MartinHjelmare Sorry... I was short sighted....
The change is in a totally different component. In that case, I do agree with you!

(Sorry @cisasteelersfan).

**Review by MartinHjelmare — APPROVED** [review]
